### ISSUE TITLE
Publish package to npm

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/atom-language-asciidoc#181]
## Description

Publish the package to npm so that it can be used with the highlights syntax highlighter in headless / library mode.

This seems to be a simple matter of running `npm publish` during a release. However, we may want to consider some excludes (using an .npmignore file). We should also qualify the package name as `atom-language-asciidoc` since we're publishing outside of the Atom ecosystem. (Another option for the name is `highlights-language-asciidoc` since technically it is not coupled to Atom).

For an example of another language, see https://www.npmjs.com/package/atom-language-nginx. I can't find a project that contains a script that shows how they are publishing to npm and apm w/ different package names.

## Code example

If this package were published to npm, you could install packages using:

```
npm install highlights
npm install atom-language-asciidoc
```

Then use it as follows:

```js
Highlights = require('highlights');
highlighter = new Highlights();
highlighter.requireGrammarsSync({
  modulePath: require.resolve('language-asciidoc/package.json')
});
html = highlighter.highlightSync({
  fileContents: '= Document Title\nAuthor Name\n\ncontent',
  scopeName: 'source.asciidoc'
});
console.log(html);
```

**Comment by ldez** [comment]
Have you a real use case ?

**Comment by mojavelinux** [comment]
Yes. I plan to integrate highlights as the convert-time syntax highlighter in Asciidoctor.js.

**Comment by ldez** [comment]
Can you create a Org on NPM for Asciidoctor ?
[Create org](https://www.npmjs.com/org/create?utm_source=house&utm_medium=features%20page&utm_campaign=free%20orgs&utm_term=Create%20an%20Org)

**Comment by mojavelinux** [comment]
We already have one. I'll add you.

**Comment by mojavelinux** [comment]
Done.
